sw/host/opentitanlib/src/image/image.rs

```diff
@@ -150,6 +143,16 @@ impl Image {
     pub fn compute_digest(&self) -> sha256::Sha256Digest {
         sha256::sha256(&self.data.bytes[offset_of!(Manifest, usage_constraints)..self.size])
     }
+
+    /// Operates on the SPHINCS+ signed region.
+    ///
+    /// Used when computing the SPHINCS+ signature.
+    pub fn map_sphincs_signed_region<F, R>(&self, f: F) -> R
+    where
+        F: FnOnce(&[u8]) -> R
+    {
+        f(&self.data.bytes[offset_of!(Manifest, rsa_signature)..self.size])
```

Comment:
I thought in recent discussions we decided that the signed region should _not_ include the `rsa_signature`.

Reply:
I'm not sure, my understanding is that `rsa_signature` is included. @alphan can you weigh in on this?

Reply:
Yeah, we decided to exclude `rsa_signature` from the signed region. Apologies for the delay in updating you @jon-flatley.